packages/extension-sdk-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@looker/extension-sdk-react",
-  "version": "21.20.1",
+  "version": "22.0.0",
```

Comment:
I am so confused - this appropriately got bumped to 22.0.0 AND, UNEXPECTEDLY, release-please (lerna) UPDATED other package's dependencies across a major breaking version... but it didn't do it for `@looker/sdk-codegen` when that went from 21.3.1 to 22.0.0 so I had to pin it back. well, hopefully this mystery is harmless...